nbbuild/antsrc/org/netbeans/nbbuild/CustomJavac.java

```diff
@@ -66,21 +66,30 @@ public void setIncludes(String includes) {
 
     @Override
     public void execute() throws BuildException {
-        String tgr = getTarget();
-        if (tgr.matches("\\d+")) {
-            tgr = "1." + tgr;
-        }
-        if (!isBootclasspathOptionUsed()) {
-            setRelease(tgr.substring(2));
-        }
-        String src = getSource();
-        if (src.matches("\\d+")) {
-            src = "1." + src;
-        }
-        if (!JavaEnvUtils.isAtLeastJavaVersion(src)) {
-            log("Cannot handle -source " + src + " from this VM; forking " + maybeFork, Project.MSG_WARN);
-            super.setFork(true);
-            super.setExecutable(maybeFork);
+        String release = getRelease();
+        if (release == null || release.isEmpty()) {
```

Comment:
Setting the `release` only when it is not set yet seems OK to me.

Reply:
thanks for taking a look jaroslav!